bitnami/postgresql-repmgr/10/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -222,13 +222,28 @@ repmgr_get_primary_node() {
         fi
     else
         if [[ -z "$upstream_host" ]]; then
+
+            info "No upstream detected: primary_host: '${REPMGR_PRIMARY_HOST}:${REPMGR_PRIMARY_PORT}'"
+            info "No upstream detected: node_network: '${REPMGR_NODE_NETWORK_NAME}:${REPMGR_PORT_NUMBER}'"
+
             if [[ "${REPMGR_PRIMARY_HOST}:${REPMGR_PRIMARY_PORT}" != "${REPMGR_NODE_NETWORK_NAME}:${REPMGR_PORT_NUMBER}" ]]; then
+
+                debug "Primary hosts differences"
+
                 primary_host="$REPMGR_PRIMARY_HOST"
                 primary_port="$REPMGR_PRIMARY_PORT"
             fi
+
+            repmgr_wait_primary_node || repmgr_switchover_standby
+
         else
+
+            info "Upstream detected: primary_host: '${upstream_host}:${upstream_port}'"
+
             primary_host="$upstream_host"
             primary_port="$upstream_port"
+
+
```

Comment:
Please remove extra whitelines

```suggestion
            info "No upstream detected: primary_host: '${REPMGR_PRIMARY_HOST}:${REPMGR_PRIMARY_PORT}'"
            info "No upstream detected: node_network: '${REPMGR_NODE_NETWORK_NAME}:${REPMGR_PORT_NUMBER}'"
            if [[ "${REPMGR_PRIMARY_HOST}:${REPMGR_PRIMARY_PORT}" != "${REPMGR_NODE_NETWORK_NAME}:${REPMGR_PORT_NUMBER}" ]]; then
                debug "Primary hosts differences"
                primary_host="$REPMGR_PRIMARY_HOST"
                primary_port="$REPMGR_PRIMARY_PORT"
            fi
            repmgr_wait_primary_node || repmgr_switchover_standby
        else
            info "Upstream detected: primary_host: '${upstream_host}:${upstream_port}'"
            primary_host="$upstream_host"
            primary_port="$upstream_port"
```

Reply:
Hi @migruiz4, 
Thanks for taking the time to check. 
I already made the changes and review your comments, Can you take a look?